### PR TITLE
Robolectric instead of androidTest

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
This PR removes need in Android Emulator, now builds on Travis CI should be fast and non-flaky!

@nikitin-da PTAL!

### CODE REVIEW

File: storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/integration/BaseTest.java [hunk 1]
```diff
@@ -33,11 +34,11 @@
 
     @Before
     public void setUp() throws Exception {
-        sqLiteOpenHelper = new TestSQLiteOpenHelper(InstrumentationRegistry.getContext());
+        sqLiteOpenHelper = new TestSQLiteOpenHelper(RuntimeEnvironment.application);
 
         db = sqLiteOpenHelper.getWritableDatabase();
 
-        storIOSQLite = new DefaultStorIOSQLite.Builder()
```
Comment (nikitin-da) [inline storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/integration/BaseTest.java]: Wow how did it works early?

**Comment by nikitin-da** [comment]
Awesome! I hope it'll solve problem once and for all

File: storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/integration/BaseTest.java (hunk 1 above)
Comment (artem-zinnatullin) [inline storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/integration/BaseTest.java]: I was surprised too, tests had same package :smile:

**Comment by artem-zinnatullin** [comment]
Yeah, me too

**Pull request merged**